lib/tbot/config/configtemplate_ssh_host_cert_test.go

```diff
@@ -84,8 +85,15 @@ func TestTemplateSSHHostCertRender(t *testing.T) {
 	err = template.Render(context.Background(), mockBot, ident, dest)
 	require.NoError(t, err)
 
+	// Make sure a cert is written. We just use a dummy cert (the CA fixture)
 	certBytes, err := memory.Read(template.Prefix + "-cert.pub")
```

Comment:
nit: Use `sshHostCertSuffix` instead of `"-cert.pub"`, similar to `sshHostCASuffix` below